docker-compose.yml

```diff
@@ -3,6 +3,7 @@ version: "3.7"
 services:
   gateway:
     image: openmrs/openmrs-reference-application-3-gateway:${TAG:-nightly}
+    build: ./gateway
```

Comment:
I'd prefer we leave these out. Since we're not using docker-compose to actually build this, this file basically serves as a reference for implementations.

Reply:
Well, actually one could use docker-compose.yml to build the production image and docker-compose.override.yml to build the dev image, but it's not setup this way yet so it's fair to leave it out for now. See e.g. https://github.com/openmrs/openmrs-core/blob/master/docker-compose.override.yml#L20-L24 and https://github.com/openmrs/openmrs-core/blob/master/docker-compose.yml#L28-L30
The dev image has all sources included whereas the production image has just binaries. I can imagine we will benefit from that in O3 at some point as well.